backend/tasks/ActivityMonitor.js

```diff
@@ -180,7 +181,7 @@ async function ActivityMonitor(interval) {
       let ExistingDataToUpdate = [];
 
       //for each item in playbackToInsert, check if it exists in the recent playback activity and update accordingly. insert new row if updating existing exceeds the runtime
-      if (playbackToInsert.length > 0 && ExistingRecords.length > 0) {
+      if (playbackToInsert.length >= MINIMUM_SECONDS_TO_INCLUDE_PLAYBACK && ExistingRecords.length >= MINIMUM_SECONDS_TO_INCLUDE_PLAYBACK) {
```

Comment:
Changed the comparisons to greater-or-equal-than, so the minimum value for the new variable could be 1 rather than 0.